cdap-proto/src/main/java/co/cask/cdap/proto/security/Authorizable.java

```diff
@@ -190,9 +190,11 @@ private static void checkParts(EntityType entityType, List<String> parts, int in
     switch (entityType) {
       case INSTANCE:
       case NAMESPACE:
-        entityParts.put(entityType, parts.get(index));
-        break;
       case KERBEROSPRINCIPAL:
+        if (parts.size() != 1 && index == (parts.size() - 1)) {
+          throw new IllegalArgumentException("Entity name should be in the format: <entity-name>, " +
```

Comment:
The format should be entity-type:entity-name. 
If you are just talking about the value then I think we should not say format.